### PR TITLE
Post calls

### PR DESCRIPTION
**Pull request by zliibbe**
# Description

This pull request does not fully close any issues as it is meant to serve as a redirect to create two more clearly defined branches to advance development. 

This pull request starts the work of allowing a user to input new data. In writing the code for this PR, I realized I needed to have a way to get information from the user displayed on the DOM, so I made 3 input boxes for sleep, hydration, and activity. These are currently on the bottom of the page, but will, in the future, effectively be their own page via button click. The plan is for a user to click a button to add data, which will hide all the data displays and show all the data entry forms. This PR shows both data and data entry form on the same page. In working through how to do Post Calls I realized we will need to separate data and DOM manipulation. This branch is being merged back in to main in order to create 2 new branches. These two branches will be called something like. postCallsDOM and postCallsData.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
    - Changed: 
    - Bugs created:
- [ ] Bug fix (non-breaking change which fixes an issue)
    -Bug fixed:
- [ ] Refactor
    -Code refactored:
- [ ] Documentation update

## How Has This Been Tested?

Display of data entry forms and data display forms were tested with verification of correct display in the browser.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have fully tested my code
- [x] My code follows the style guidelines of this project.
- [ ] My code has no unused/commented-out code.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

## Future Steps:

Separate the post-call functionality into two new branches; one branch for addressing how the information is dealt with for post calls and one branch for how that data is displayed on the DOM.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -81,8 +85,48 @@ <h1>HealthDash</h1>
...
+              <label for="flightsOfStairs">Flights of stairs:</label>
+                <input type="number">
+              <label for="minutesActive">Minutes Active:</label>
+                <input type="number">
+                <input type="submit" class="submit-activity">
+            </form>
+        </section>
+      </section>
+      <button class="back-to-main">Back to Main</button>
+    </main>
```
Comment (Andrew-Musselman) [inline dist/index.html:130]: The forms look good!

File: src/css/styles.css [hunk 2]
```diff
@@ -149,3 +183,54 @@ main {
...
+}
+
+.back-to-main {
+  margin-left: 45%;
+  width: 20vh;
+  height: 5vh;
+  justify-content: center;
+  align-content: center;
+  flex-flow: row wrap;
+}
```
Comment (Andrew-Musselman) [inline src/css/styles.css:236]: Looks good!

**Review by zliibbe — COMMENTED** [review]
Looks like appropriate changes considering our separation of DOM and data. Thanks for this review Andrew.

**Review by Andrew-Musselman — APPROVED** [review]
Everything looks good on my end. The merge conflict was easy to resolve

**Pull request merged**